drivers/telescope/lx200ap_experimental.cpp

```diff
@@ -591,12 +710,85 @@ bool LX200AstroPhysicsExperimental::ISNewSwitch(const char *dev, const char *nam
 
 bool LX200AstroPhysicsExperimental::ReadScopeStatus()
 {
+    double lng = LocationN[LOCATION_LONGITUDE].value;
+    double lst = get_local_sidereal_time(lng);
+    // 2020-06-02, wildi, isParked is reserved for the state in ParkData.xml
+    // see method isParked()
+    bool isAPParked ;
+    getMountStatus(&isAPParked);
+    if (!isAPParked)
+    {
+        // in case of simulation, the coordinates are set on parking
+        HourangleCoordsNP.s = IPS_BUSY;
+        IDSetNumber(&HourangleCoordsNP, nullptr);
+        HourangleCoordsN[0].value = get_local_hour_angle(lst, currentRA);
+        HourangleCoordsN[1].value = currentDEC;
+        HourangleCoordsNP.s = IPS_OK;
+        IDSetNumber(&HourangleCoordsNP, nullptr );
+    }
+    double val;
+    if ((!isSimulation()) && (getSDTime(PortFD, &val) < 0))
+    {
+        LOG_ERROR("Reading sidereal time failed %d");
+        return false;
+    }
+    if (isSimulation())
+    {
+        val = lst;
+    }
+    APSiderealTimeNP.np[0].value = val;
+    APSiderealTimeNP.s           = IPS_IDLE;
+    IDSetNumber(&APSiderealTimeNP, nullptr);
+
     if (isSimulation())
     {
         mountSim();
         return true;
     }
 
+    if (getAPUTCOffset(PortFD, &val) < 0)
+    {
+        LOG_ERROR("Reading UTC offset  %d");
+    }
+    if (getLocalTime24(PortFD, &val) < 0)
+    {
+        LOGF_DEBUG("Reading local time failed :GL %d", -1);
```

Comment:
Shouldn't this be LOG_ERROR or LOG_WARN at least?

Reply:
Yes and no: this is in ReadScopeStatus(). There is an error message if UTC offset can not be read, hence the user is informed. I recommend to leave it. 

I changed LOGF_* to LOG_*